CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+## 0.4.0 (July 17, 2019)
+FEATURES:
+* [24](https://github.com/nginxinc/nginx-plus-go-client/pull/24): *Support `MaxConns` in upstream servers*.
+
+BUGFIXES:
+* [25](https://github.com/nginxinc/nginx-plus-go-client/pull/25): *Fix session metrics for stream upstreams*. Session metrics with a status of `4xx` or `5xx` are now correctly reported. Previously they were always reported as `0`.
```

Comment:
I suggest renaming the corresponding PR as well. Sessions are part of stream server zone not stream upstreams.
```suggestion
* [25](https://github.com/nginxinc/nginx-plus-go-client/pull/25): *Fix session metrics for stream server zones*. Session metrics with a status of `4xx` or `5xx` are now correctly reported. Previously they were always reported as `0`.
```